docs/Tutorials/CCE.md

```diff
@@ -0,0 +1,238 @@
+    ax.set_ylabel("Mode coefficient")
+    ax.set_title(plot_quantities[i])
+plt.tight_layout()
+plt.savefig(output_plot_filename, dpi=400)
+plt.clf()
+```
+
+### Input data formats
+
+The worldtube data must be constructed as spheres of constant coordinate
+radius, and (for the time being) written to a filename of the format
+`...CceRXXXX.h5`, where the `XXXX` is to be replaced by the integer for which
+the extraction radius is equal to `XXXX`M. For instance, a 100M extraction
+should have filename `...CceR0100.h5`. This scheme of labeling files with the
+extraction radius is constructed for compatibility with SpEC worldtube data.
+
+There are two possible formats of the input data, one based on the Cauchy metric
+at finite radius, and one based on Bondi data. The metric data format must be
+provided as spherical harmonic modes with the following datasets:
+- `gxx.dat`, `gxy.dat`, `gxz.dat`, `gyy.dat`, `gyz.dat`, `gzz.dat`
+- `Drgxx.dat`, `Drgxy.dat`, `Drgxz.dat`, `Drgyy.dat`, `Drgyz.dat`, `Drgzz.dat`
+- `Dtgxx.dat`, `Dtgxy.dat`, `Dtgxz.dat`, `Dtgyy.dat`, `Dtgyz.dat`, `Dtgzz.dat`
+- `Shiftx.dat`, `Shifty.dat`, `Shiftz.dat`
+- `DrShiftx.dat`, `DrShifty.dat`, `DrShiftz.dat`
+- `DtShiftx.dat`, `DtShifty.dat`, `DtShiftz.dat`
+- `Lapse.dat`
+- `DrLapse.dat`
+- `DtLapse.dat`
+
+In this format, each row must start with the time stamp, and the remaining
+values are the complex modes in m-varies-fastest format.
+
+The second format is Bondi-Sachs metric component data.
+This format is far more space-efficient (by around a factor of 4), and SpECTRE
+provides a separate executable for converting to the Bondi-Sachs worldtube
+format, `ReduceCceWorldtube`.
+The `ReduceCceWorldtube` executable should be run on a Cauchy metric worldtube
+file, and will produce a corresponding 'reduced' Bondi-Sachs worldtube file.
+The basic command-line arguments for the executable are:
+```
+ReduceCceWorldtube --input-file CceR0050.h5 --output-file BondiCceR0050.h5\
+ --lmax_factor 3
+```
+The argument `--lmax_factor` determines the factor by which the resolution at
+which the boundary computation that is run will exceed the resolution of the
+input and output files.
+Empirically, we have found that `lmax_factor` of 3 is sufficient to achieve
+roundoff precision in all boundary data we have attempted, and an `lmax_factor`
+of 2 is usually sufficient to vastly exceed the precision of the simulation that
+provided the boundary dataset.
+
+The format is similar to the metric components, except in spin-weighted
+spherical harmonic modes, and the real (spin-weight-0) quantities omit the
+redundant negative-m modes and imaginary parts of m=0 modes.
+The quantities that must be provided by the Bondi-Sachs metric data format are:
+- `Beta.dat`
+- `DrJ.dat`
+- `DuR.dat`
+- `H.dat`
+- `J.dat`
+- `Q.dat`
+- `R.dat`
+- `U.dat`
+- `W.dat`
+
+The Bondi-Sachs data may also be used directly for CCE input data.
+The input option to specify the input data type is `Cce.H5IsBondiData`,
```

Comment:
Maybe give a glimpse of the input file in a code block? Like you did for `NoIncomingRadiation`